tools/rosbash/rosfish

```diff
@@ -22,7 +22,7 @@ function _rosfind
         # BSD find needs -E for extended regexp
         find -E $argv
     else
-        find $argv
+        find $arg
```

Comment:
@iPaddy Excuse me, why was this changed? I believe `$arg` is empty here and `$argv` is correct.

How to confirm:

```sh
# Create directory to avoid slow `find`
mkdir tmp
cd tmp

# Shows no candidate
rosrun rviz <Tab>

# Edit `find $arg` to `find $argv`
sudo vim /opt/ros/noetic/share/rosbash/rosfish

# Correctly be complemented
rosrun rviz <Tab>
```

Reply:
I created PR.

https://github.com/ros/ros/pull/279